src/Kunstmaan/AdminBundle/Resources/ui/js/_auto-collapse-tabs.js

```diff
@@ -18,6 +21,44 @@ kunstmaanbundles.autoCollapseTabs = (function($, window, undefined) {
         }); // when window is resized
     };
 
+    initTabLogic = function () {
+        // If there is a tab defined in the url, we activate it
+        if ($('#currenttab') && $('#currenttab').val().length > 0) {
+            $('.js-auto-collapse-tabs.nav-tabs a[href="' + $('#currenttab').val() + '"]').tab('show');
+        }
+
+        // When tab click, add the current tab in the url
+        $('.js-auto-collapse-tabs.nav-tabs a').click(function (e) {
+            $(this).tab('show');
+
+            var activeTab = this.hash.substr(1);
+            window.history.pushState({}, null, replaceUrlParam(window.location.href, 'currenttab', activeTab));
```

Comment:
history.pushState is not supported in ie9. I'll look into another way of achieving this.